### ISSUE TITLE
Enhance the content of official examples

### ISSUE THREAD

**Issue by mitmul** [chainer/chainer#3799]
We need to add more examples to the official repo. This issue is for listing up the candidates.
Thanks for providing the list below, @unnonouno @rezoo @mattya @delta2323 

Location for example should be either this repo, or pfnet-research, which will be labelled as external.

## NLP
Lead: @unnonouno 

- Sequential labeling #1375 Merged!
- Document classification #3029 Merged!
- Memory networks #4222 Merged!
- seq2seq with attention  #4123 WIP?
- Visual Question Answering ?
- Mini-batch training for recursive neural networks (https://github.com/chainer/chainer/pull/2135) @unnonouno @beam2d Next step?

## Computer Vision
Lead: @rezoo 

- Depth estimation using CNN @t-abe 
- Optical Flow using CNN
- Style transfer @rezoo 
- Image captioning @hvy #4076 Merged!
- Super-resolution
- Video interpolation
- Binary CNN
- Pose estimation using CNN (OpenPose?) @mitmul
- Unsupervised image representation learning

## Generative models
Lead: @mattya 

- conditional GAN (with spectral normalization?) @takerum to provide after 1/29
- WGAN-GP (link ext. to Chainer-GAN-lib?) @mattya 
- text to image (complicated?)
- semi-supervised learning with GAN
- pix2pix (inpainting, colorization, etc.) @taizan 
- cycle GAN asked @rcalland 
- deep dream
- 3d model GAN
- wavenet
- Unsupervised Learning by Predicting Noise
- Energy based models

## Bio
Lead: @delta2323 
- Protein secondary structure prediction with cascaded CNN and RNN (https://github.com/delta2323/BMI219-2017-ProteinFolding) -- please copy repo to pfnet-research
- Multitask learning with multi layer perceptron for QSAR (https://github.com/delta2323/BMI219-2017-DeepQSAR) -- please copy repo to pfnet-research

## Reinforcement Learning
Lead: ??
- Neural Architecture Search

## Others

- Capsule network #4061
- Double Backprop Example ?

**Comment by hvy** [comment]
As for **image captioning**, @soskek san has a repository [here](https://github.com/soskek/captioning_chainer) and I also implemented a very similar model [here](https://github.com/hvy/chainer-image-caption-vg).

**Comment by hvy** [comment]
Do we have any guidelines for whether or not training must be supported, or how to use/reuse datasets that are not yet wrapped by Chainer?

**Comment by Crissman** [comment]
@hvy These are meant to be examples, so a variety of approaches is fine. Feel free to use the trainer or not, whichever is your style.

**Comment by Crissman** [comment]
@hvy For any examples that will be included in the Chainer repo, I think we should use wrapped datasets. For separate repositories, ok to include instructions on how to get the data set.

**Comment by hvy** [comment]
Image captioning example being worked on https://github.com/chainer/chainer/pull/4076

**Comment by unnonouno** [comment]
I sent memory network example on #4222

**Comment by delta2323** [comment]
Sorry for waiting too long. I have moved my repositories to pfnet-research organization. Should we change the repository names?

https://github.com/pfnet-research/BMI219-2017-DeepQSAR
https://github.com/pfnet-research/BMI219-2017-ProteinFolding

**Comment by mitmul** [comment]
This issue will be addressed by the new project here: https://github.com/chainer/models
